### ISSUE TITLE
plugin still seems to use old xtend stuff

### ISSUE THREAD

**Issue by cdietrich** [xtext/xtext-gradle-plugin#193]
```
12:43:40  FAILURE: Build failed with an exception.
12:43:40  
12:43:40  * What went wrong:
12:43:40  A problem occurred configuring root project 'xtext-core'.
12:43:40  > Could not resolve all artifacts for configuration ':classpath'.
12:43:40     > Could not resolve com.google.guava:guava:[10.0.1,14.0.1].
12:43:40       Required by:
12:43:40           project : > org.xtext:xtext-gradle-plugin:2.1.0 > org.eclipse.xtend:org.eclipse.xtend.lib:2.9.0 > org.eclipse.xtext:org.eclipse.xtext.xbase.lib:2.9.0
12:43:40           project : > org.xtext:xtext-gradle-plugin:2.1.0 > org.eclipse.xtend:org.eclipse.xtend.lib:2.9.0 > org.eclipse.xtend:org.eclipse.xtend.lib.macro:2.9.0
12:43:40        > Could not resolve com.google.guava:guava:14.0.1.
12:43:40           > Could not get resource 'https://plugins.gradle.org/m2/com/google/guava/guava/14.0.1/guava-14.0.1.pom'.
12:43:40              > Could not HEAD 'https://jcenter.bintray.com/com/google/guava/guava/14.0.1/guava-14.0.1.pom'.
12:43:40                 > Read timed out
12:43:40  
```

https://ci.eclipse.org/xtext/job/xtext-core/job/cd_fixJavadoc/4/console

am not sure if this is an order problem or side effect of comm problem from eclipse infrastructure etc.
we also dont have any jcenter refs. wonder if https://plugins.gradle.org/m2 sends some bad redirects  (https://status.gradle.com/incidents/ndfqcnmqc8n7)

anyway i would expected xbase.lib 2.25.0.M1 to be pulled

**Comment by sergio-otero** [comment]
It was supposed to close down on February 3 2021 (https://blog.gradle.org/jcenter-shutdown) but gradle is saying it's an issue

https://status.gradle.com/

> Update - We are continuing to work on a fix for this issue.
> Sep 16, 09:54 UTC
> Identified - JCenter response time is currently very slow and affects operations on the Gradle plugin portal.
> Sep 16, 09:53 UTC

I also have problems with an XText Eclipse dependency of a custom plugin not depending on yours, and my build script does not depend directly on jCenter ...

> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve org.eclipse.emf:org.eclipse.emf.common:[2.10.1,3).
     Required by:
         project : > xxx-plugin:2.3.2 > yyy:2.3.2 > org.eclipse.xtext:org.eclipse.xtext:2.13.0 > org.eclipse.xtext:org.eclipse.xtext.util:2.13.0
      > Could not resolve org.eclipse.emf:org.eclipse.emf.common:2.23.0.
         > Could not get resource 'https://plugins.gradle.org/m2/org/eclipse/emf/org.eclipse.emf.common/2.23.0/org.eclipse.emf.common-2.23.0.pom'.
            > Could not HEAD 'https://jcenter.bintray.com/org/eclipse/emf/org.eclipse.emf.common/2.23.0/org.eclipse.emf.common-2.23.0.pom'.
               > Read timed out

**Comment by oehme** [comment]
The plugin itself is implemented in Xtend 2.9. So this lookup was expected. The problem was a temporary issue on the Gradle plugin portal.
